whylog/teacher/__init__.py

```diff
@@ -159,6 +140,20 @@ def set_causes_relation(self, relation):
         """
         pass
 
+    def make_group(self, line_id, span):
```

Comment:
We should ask Client if we need it and if yes, we can create proper mechanism in Front. I suggest not to remove it yet.